src/http/response.ts

```diff
@@ -50,6 +53,9 @@ export enum LkErrorKey {
   CRITICAL = 'critical',
   BUG = 'bug',
   SOCKET_ERROR = 'socket_error',
+  PORT_BUSY = 'port_busy',
+  PORT_RESTRICTED = 'port_restricted',
```

Comment:
Looks like these two error keys are currently unused. However, they should be used in https://github.com/Linkurious/linkurious-server/blob/493cf372002e28a13580298d58894eb299b693cb/server/services/webServer/index.ts#L391-L410, so let's add them here.